mailpoet/views/form/editor.html

```diff
@@ -103,6 +103,10 @@
   'formPlacementPlacementPositionLeft': _x('left', 'Placement of a slide in form, on the left or right side of the page'),
   'formPlacementPlacementPositionRight': _x('right', 'Placement of a slide in  form, on the left or right side of the page'),
   'formPlacementDelaySeconds': _x('%1s sec', 'times selection should be in the end "30 sec"'),
+  'formPlacementCookieExpiration': _x('Show form every', 'Label on a selection of different times'),
+  'formPlacementCookieExpirationAlways': _x('Always', 'times selection'),
```

Comment:
I would have liked to use the [_nx](https://developer.wordpress.org/reference/functions/_nx/) function but didn't seem to work with our translation setup.

Reply:
You're probably right. Even if you add it to template [i18n functions](https://github.com/mailpoet/mailpoet/blob/master/mailpoet/lib/Twig/I18n.php#L29), our makepot [extractor](https://github.com/mailpoet/mailpoet/blob/master/mailpoet/tasks/makepot/extract.php) may not be able to pick it up. And rewriting its regular expressions is a task not many would dare to do.